examples/tut-6-rpc/client-py/index.py

```diff
@@ -0,0 +1,108 @@
+import json
+import os
+import random
+import requests
+from concurrent.futures import ALL_COMPLETED, ThreadPoolExecutor, wait
+from time               import sleep
+
+
+def log(t):
+    print(t, flush=True)
```

Comment:
Just use `print`... I don't want to have people think that logging fancy is needed in the example

Reply:
`flush=True` is necessary unless you want all output from the Python client to be buffered and spat to console in one go, unfortunately. Raw `print` also prevents interleaving output with the server's logging.

Reply:
Oh, maybe I misunderstood you - are you asking me to replace `log` with `print` + inline the extra `flush=True` everywhere?

Reply:
I'm just saying to just replace `s/log/print` and don't worry about the downsides... because the point of this code is to be beautiful for someone to say, "Wow, look how simple using Reach is from Python". Every little bit of friction counts

Reply:
De-logged in 3cd5ef1.